check-log/check-log.go

```diff
@@ -361,8 +361,8 @@ func writeFileAtomically(f string, contents []byte) error {
 		return err
 	}
 	defer os.Remove(tmpf.Name())
-	defer tmpf.Close()
 	_, err = tmpf.Write(contents)
+	tmpf.Close() // should be called before rename
```

Comment:
Ah, right. The file handle should be closed before Rename. Especially this will be problem on Windows. Sorry. :bowing_man: